src/main/java/be/syntra/devshop/DevshopFront/models/SaveStatus.java

```diff
@@ -0,0 +1,17 @@
+package be.syntra.devshop.DevshopFront.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
```

Comment:
Replace 'AllArgsConstructor' by 'RequiredArgsConstructor'.  
Make the label field final
Remove the label function (you basically created a getLabel() with extra steps)